en/engine/animation/skeletal-animation.md

```diff
@@ -0,0 +1,73 @@
+
+This provides the most basic support for all the following functions:
+  * blendshape support
+  * Mixing and masking of any number of animation clips
+  * IK, secondary physical influence
+  * Pure program control joint position
+
+## Selection and best practice of two systems
+
+After importing all model resources, all prefabs use the pre-baking system by default to achieve the best performance. It is recommend that you only use the real-time computing system if you clearly feel that the performance of the pre-baking system cannot reach the standard. Although the two systems can be switched seamlessly at runtime, try not to perform high frequency, because each switch involves the reconstruction of the underlying rendering data.
+
+## Skinning algorithm
+
+We have built-in two common standard skinning algorithms, which have similar performance and only affect the final performance:
+
+1. LBS (Linear Mixed Skinning): bone information is stored in the form of a 3x4 matrix, and the matrix is ​​interpolated directly to achieve skinning, and there are typical known problems such as volume loss.
+2. DQS (Dual Quaternion Skinning): The bone information is interpolated in the form of double quaternion, which is more accurate and natural for the skeleton animation without scaling transformation, but for performance reasons, there is an approximate simplified process for all scaling animations.
+
+The engine uses LBS by default. You can switch the skinning algorithm by modifying the `updateJointData` function reference of the engine skeletal-animation-utils.ts and the header file reference in cc-skinning.chunk.
+
+It is recommended that projects with higher pursuit of skin animation quality can try to enable DQS, but since there is no `fma` instruction before GLSL 400, operations such as `cross` cannot bypass floating-point cancellation on some GPUs, and the error is relatively high. Large, may introduce some visible defects.
+
+## Hanging point system
+
+If you need to hang some external nodes to the specified bone joints, you need to use the Socket system of the skeleton animation component:
+  * Create a new child node under the bone animation component to be docked (the parent directly should be the node where the animation component is located).
+  * Add an array element to the sockets property of the skeleton animation component, select the path of the bone to be connected from the drop-down list, and specify the target as the child node just created.
+  * This child node becomes the target hanging point, you can put any external node under this child node, it will follow the specified bone transformation.
+
+The hanging point model in the `FBX` or `glTF` resource will automatically connect to the hanging point system without any manual operation.
```

Comment:
`FBX` or `glTF` resources will be automatically adapted to use this socket system at import time, without any manual interference.